Makefile

```diff
@@ -3,17 +3,21 @@ obj-m += mtcadummy.o
 #functionality (mtcadummy_drv) and the proc file debug output (mtcadummy_proc)
 mtcadummy-objs := mtcadummy_drv.o mtcadummy_proc.o mtcadummy_firmware.o mtcadummy_rw_no_struct.o
 
-KVERSION = $(shell uname -r)
+KVERSION := $(shell uname -r)
 
 #define the package/module version
 #DO NOT FORGET TO ADAPT BOTH DRIVER AND MODULE VERSION!
 MTCADUMMY_MAJOR_VERSION=0
 MTCADUMMY_MINOR_VERSION=10
 MTCADUMMY_PACKAGE_VERSION=0.10.2
```

Comment:
This duplicates the version information from the CMakeLists.txt. There should be a single source of information.

Reply:
The CMake is only for fooling Jenkins into building this, so I suppose the version can go from that